code/modules/mob/mob_defines.dm

```diff
@@ -42,6 +42,7 @@
 	var/ear_damage = 0		//Carbon
 	var/stuttering = 0		//Carbon
 	var/slurring = 0		//Carbon
+	var/cultslurring = 0	//Carbon
```

Comment:
To prevent that I could force the slur to 0 when it's positive. However, the code to do that would look really ugly. I think having an extra mob var is better than messing with positive and negative vars.